src/style/modal.scss

```
@@ -0,0 +1,392 @@
+.modal{
+    display: none;
+
+    position: fixed;
+    left: 18%;
+    top: 0;
+    width: 1025px;
+    height: 100vh;
+    z-index: 9998;
+}
+.modal__box {
+    max-width: 100%;
+    padding: 30px 30px;
+    z-index:1;
+    background-color: white;
+    //margin: 30px 15px;
+    padding: 20px;
+
+    display:flex;
+    flex-direction: column;
+    
+}
+.open{
+    display:block;
+}
+.wrapper-for-box{
+    display:flex;
+    gap:30px;
+    padding: 50px;
+
+}
+.inputItems__date-time{
+    display:flex;
+    gap:20px;
+}
+#time, #date{
+    width: 192px;
+    height: 45px;
+}
+.close {
+    display:flex;
+    align-items: flex-end;
+    justify-content: end;
+    img{
+        cursor: pointer;
+    }
+}
+.modal__box__first {
+    display: flex;
+    flex-direction: column;
+    gap:35px;
+
+}
+.moda__box__first__h {
+    display:flex;
+    flex-direction: column;
+}
+.moda__box__first__h__img {
+}
+.moda__box__first__h__text {
+    font-size: 50px;
+    font-family: $Forum;
+    color:$colorH;
+}
+.moda__box__first__h2 {
+    display:flex;
+    flex-direction: column;
+    gap:15px;
+}
+.moda__box__first__h2__text {
+    font-family: $Roboto;
+    font-size: 18px;
+    color: $lineColor;
+    font-weight: bold;
+}
+.moda__box__first__h2__line {
+    width: 100%;
+    height: 3px;
+    background-color: $lineColor;
+}
+.inputItems__entryTicket{
```

kebab-case + форматирование
```suggestion
.inputItems__entry-ticket {
```